File: bitget-java-sdk-api/src/main/java/com/bitget/openapi/api/v2/MixAccountApi.java
package com.bitget.openapi.api.v2;

import com.bitget.openapi.dto.response.ResponseResult;
import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.GET;
import retrofit2.http.POST;
import retrofit2.http.QueryMap;

import java.util.Map;

public interface MixAccountApi {

    @GET("/api/v2/mix/account/account")
    Call<ResponseResult> account(@QueryMap Map<String, String> paramMap);

    @GET("/api/v2/mix/account/accounts")
    Call<ResponseResult> accounts(@QueryMap Map<String, String> paramMap);

    @POST("/api/v2/mix/account/set-leverage")
    Call<ResponseResult> setLeverage(@Body Map<String, String> paramMap);

    @POST("/api/v2/mix/account/set-margin")
    Call<ResponseResult> setMargin(@Body Map<String, String> paramMap);

    @POST("/api/v2/mix/account/set-margin-mode")
    Call<ResponseResult> setMarginMode(@Body Map<String, String> paramMap);

    @POST("/api/v2/mix/account/set-position-mode")
    Call<ResponseResult> setPositionMode(@Body Map<String, String> paramMap);


    // position
    @GET("/api/v2/mix/position/single-position")
    Call<ResponseResult> singlePosition(@QueryMap Map<String, String> paramMap);

    @GET("/api/v2/mix/position/all-position")
    Call<ResponseResult> allPosition(@QueryMap Map<String, String> paramMap);
}

File: bitget-node-sdk-api/src/lib/v2/MixAccountApi.ts
import {BaseApi} from '../BaseApi';

export class MixAccountApi extends BaseApi {

    account(qsOrBody: object) {
        const url = '/api/v2/mix/account/account';
        const headers = this.signer('GET', url, qsOrBody)
        return this.axiosInstance.get(url, {headers, params: qsOrBody})
    }

    accounts(qsOrBody: object) {
        const url = '/api/v2/mix/account/accounts';
        const headers = this.signer('GET', url, qsOrBody)
        return this.axiosInstance.get(url, {headers, params: qsOrBody})
    }

    setLeverage(qsOrBody: object) {
        const url = '/api/v2/mix/account/set-leverage';
        const headers = this.signer('POST', url, qsOrBody)
        return this.axiosInstance.post(url, qsOrBody, {headers})
    }

    setMargin(qsOrBody: object) {
        const url = '/api/v2/mix/account/set-margin';
        const headers = this.signer('POST', url, qsOrBody)
        return this.axiosInstance.post(url, qsOrBody, {headers})
    }

    setMarginMode(qsOrBody: object) {
        const url = '/api/v2/mix/account/set-margin-mode';
        const headers = this.signer('POST', url, qsOrBody)
        return this.axiosInstance.post(url, qsOrBody, {headers})
    }

    setPositionMode(qsOrBody: object) {
        const url = '/api/v2/mix/account/set-position-mode';
        const headers = this.signer('POST', url, qsOrBody)
        return this.axiosInstance.post(url, qsOrBody, {headers})
    }

    singlePosition(qsOrBody: object) {
        const url = '/api/v2/mix/position/single-position';
        const headers = this.signer('GET', url, qsOrBody)
        return this.axiosInstance.get(url, {headers, params: qsOrBody})
    }

    allPosition(qsOrBody: object) {
        const url = '/api/v2/mix/position/all-position';
        const headers = this.signer('GET', url, qsOrBody)
        return this.axiosInstance.get(url, {headers, params: qsOrBody})
    }

}